src/syft/ast/globals.py

```diff
@@ -7,31 +8,24 @@
 # syft relative
 from .callable import Callable
 from .module import Module
-from .util import unsplit
 
 
 class Globals(Module):
-
-    _copy: Optional["copyType"]
     """The collection of frameworks held in the global namespace"""
 
-    def __init__(self) -> None:
-        super().__init__("globals")
-
     def __call__(
         self,
-        path: Union[str, List[str]] = [],
+        path: Optional[List[str]] = None,
         index: int = 0,
-        return_callable: bool = False,
-        obj_type: Optional[type] = None,
     ) -> Optional[Union[Callable, CallableT]]:
-        if isinstance(path, str):
-            path = path.split(".")
-        return self.attrs[path[index]](
-            path=path,
+
+        _path: List[str] = (
+            path.split(".") if isinstance(path, str) else path if path else []
```

Comment:
Q: Shouldn't we raise an error if ```path``` is not available? (here we use an empty list, but we would return an error on line 26 when we try to access it)